Content.Shared/Throwing/ThrownItemSystem.cs

```diff
@@ -54,17 +55,19 @@ private void OnHandleState(EntityUid uid, ThrownItemComponent component, ref Com
 
         private void ThrowItem(EntityUid uid, ThrownItemComponent component, ThrownEvent args)
         {
-            if (!EntityManager.TryGetComponent(component.Owner, out PhysicsComponent? physicsComponent) ||
-                physicsComponent.Fixtures.Count != 1) return;
+            if (!EntityManager.TryGetComponent(component.Owner, out FixturesComponent? fixturesComponent) ||
+                fixturesComponent.Fixtures.Count != 1) return;
+            if (!EntityManager.TryGetComponent(component.Owner, out PhysicsComponent? physicsComponent)) return;
 
-            if (_fixtures.GetFixtureOrNull(physicsComponent, ThrowingFixture) != null)
+            if (fixturesComponent.Fixtures.ContainsKey(ThrowingFixture))
             {
                 Logger.Error($"Found existing throwing fixture on {component.Owner}");
                 return;
             }
-
-            var shape = physicsComponent.Fixtures[0].Shape;
-            _fixtures.CreateFixture(physicsComponent, new Fixture(physicsComponent, shape) {CollisionLayer = (int) CollisionGroup.ThrownItem, Hard = false, ID = ThrowingFixture});
+            var fixture = fixturesComponent.Fixtures.Values.First();
+            var shape = fixture.Shape;
+            var throwingFixture = new Fixture(physicsComponent, shape) { CollisionLayer = (int) CollisionGroup.ThrownItem, Hard = false, ID = ThrowingFixture };
+            _fixtures.CreateFixture(physicsComponent, throwingFixture, true, fixturesComponent);
```

Comment:
why set update to true? just use a named parameter here if you want to pass fixturesComponent, like so:
```suggestion
            _fixtures.CreateFixture(physicsComponent, throwingFixture, manager: fixturesComponent);
```